### PR TITLE
Fix/Aut-97/Text-with-sub-superscript-doesn't-fit-borders

### PR DESCRIPTION
**Pull request by focaloid-dev**
### **Related issue:**
https://oat-sa.atlassian.net/browse/AUT-97

### **Description**

During the creation Choice interaction, after the adding text with subscript and superscript, this text overlap each other, and on the test preview page this text doesn't fit the borders

**Steps to reproduce:**

1. Add the Select Choice interaction to the canvas
2. Change the choice label with subscript/superscript text
3. Create a test with the item from steps 1-2
4. Preview the test

**Actual result**: Text is moved up from the checkbox height and doesn't fit the borders
**Expected result**:Text is on the same height level with the checkbox of choice interaction

### **Environment**
http://tao-test-alshoja.playground.kitchen.it.taocloud.org:46031

### CODE REVIEW

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
looks good good job, 
let's please make a demo 
and setup a playground environment where this can be tested

File: views/scss/qti/_choice.scss
Comment (AnastasiaArcadia) [inline views/scss/qti/_choice.scss:213]: I think we need to delete the previous declaration: `justify-content: center;`

**Review by AnastasiaArcadia — APPROVED** [review]
Works fine, just a small comment

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not master

File: views/scss/qti/_choice.scss
Comment (focaloid-dev) [reply views/scss/qti/_choice.scss:213]: OK i will change that and push

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
lgtm good job,  
please wait for approval from @bziondik and validate with QA before merge
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: views/scss/qti/_choice.scss
Comment (bziondik) [reply views/scss/qti/_choice.scss:212]: Checking item render (that is used in Preview and works without style issue) found that `div.label-content` contains `div` inside that has `display:block` by default
image
In authoring inside `div[data-html-editable=true]` that has `display: flex;`. 
If you put `display:block` it will fix issue

Comment (focaloid-dev) [reply views/scss/qti/_choice.scss:212]: so i will try to change the flex to block (in line ) and remove all flex properties

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://tao-test-alshoja.playground.kitchen.it.taocloud.org:46031/

**Pull request merged**
